wodles/docker-listener/DockerListener.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+
+
+try:
+    import docker
+except:
+    print("'docker' module needs to be installed. Execute 'pip install docker' to do it.")
+    exit(1)
+import threading
+import json
+import socket
+import sys
+import time
+
+
+class DockerListener:
+
+    wait_time = 5
+    field_debug_name = "Wodle event"
+
+    def __init__(self):
+        """"
+        DockerListener constructor
+
+        """
+        # socket variables
+        if sys.platform == "win32":
+            self.wazuh_path = 'C:\Program Files (x86)\ossec-agent'
+            print("ERROR: This wodle does not work on Windows.")
+            sys.exit(1)
+        else:
+            self.wazuh_path = open('/etc/ossec-init.conf').readline().split('"')[1]
+        self.wazuh_queue = '{0}/queue/ossec/queue'.format(self.wazuh_path)
+        self.msg_header = "1:Wazuh-Docker:"
+        # docker variables
+        self.client = docker.from_env()
+        self.send_msg(json.dumps({self.field_debug_name: "Started"}))
+        self.thread1 = threading.Thread(target=self.listen)
+        self.thread2 = threading.Thread(target=self.listen)
+        self.connect(first_time=True)
+
+    def connect(self, first_time=False):
+        """
+        Connects to Docker service, retrying if connection is not successful
+
+        """
+        if self.check_docker_service():
+            if not first_time:
+                # this is for having a thread assigned to a variable ever
```

Comment:
Improve this comment: _Two threads are necessary to always keep a reference to both threads used, even though they're not used at the same time_.